app/src/main/java/com/google/android/samples/socialite/repository/ChatRepository.kt

```diff
@@ -107,6 +108,12 @@ class ChatRepository @Inject internal constructor(
         )
 
         coroutineScope.launch {
+            // Special incoming message indicating to add shorts videos to try preload in exoplayer
+            if (text == "preload") {
```

Comment:
This experience for testing is okay, but if you want to merge this, it should probably be in the main text UI as "story" type blocks that open up this screen, as this just seems like a broken user experience.